recipe/meta.yaml

```diff
@@ -58,7 +63,7 @@ requirements:
     - openjpeg
     - openssl {{ openssl }}
     - pcre2
-    - poppler
+    - poppler 22.12.0
     - proj
     # qhull disabled because of https://github.com/conda-forge/qgis-feedstock/issues/284#issuecomment-1356490896
```

Comment:
would you be able to unpin the deps that are covered in the base cbc, and replace them with `{{ ... }}`?

Reply:
Done!